spec/fontcustom/cli_spec.rb

```diff
@@ -26,5 +26,23 @@
         expect(File.read(css)).to match("../foo/bar/")
       end
     end
+
+    context 'single quotes' do
+      it "should generate fonts and templates with single quotes" do
+        live_test do |testdir|
+          Fontcustom::CLI.start ["compile", "vectors", "--templates", "preview", "css", "scss-rails", "--single-quotes"]
+          preview = File.join testdir, "fontcustom", "fontcustom-preview.html"
+
+          expect(Dir.glob(File.join(testdir, "fontcustom", "fontcustom_*\.{ttf,svg,woff,eot}")).length).to eq(4)
+          expect(File.exists?(preview)).to be_truthy
+
+          generated_css = File.read Dir.glob(File.join(testdir, "fontcustom", "*.css")).first
+          expect(generated_css.scan('"').count).to be 0
```

Comment:
Your assertions don't match the test. Can you add an assertion that confirms single quotes are present, rather than testing the lack of double quotes?